ibm/data_source_ibm_is_instance.go

```diff
@@ -441,6 +441,54 @@ func dataSourceIBMISInstance() *schema.Resource {
 					},
 				},
 			},
+			"placement_target": &schema.Schema{
```

Comment:
Docs is not updated for this feature in is_instance datasource

Since it is a list does it have to be placement_targets?

Reply:
It will have only one element. There is only one placement target. List is for having its attributes like CRN, Id etc.